verkle-trie/src/proof.rs

```diff
@@ -117,7 +116,7 @@ impl VerificationHint {
             diff_stem_no_proof,
         })
     }
-    pub fn write<W: Write>(&self, writer: &mut W) -> IOResult<()> {
+    pub fn write<W: Write>(&self, writer: &mut W) -> Result<(), std::io::Error> {
```

Comment:
Was this supposed to return a HintError instead of std::io::Error ?

Reply:
Yes, although I think next PR I'm going to change this a direct implementation of the `std` "write" trait. Tracking in #69